### PR TITLE
swap `uglifyjs` for `uglify-js`

### LINKED ISSUE #82: Cannot find module 'uglifyjs'
I get "Error: Cannot find module 'uglifyjs'" when I run npm install.  I am a new00be at npm, but it looks as if the module may be unpublished https://registry.npmjs.org/uglifyjs

Thoughts

### PR DESCRIPTION
**Pull request by fyockm**
`uglifyjs` was apparently unpublished:
https://www.npmjs.com/package/uglifyjs

`uglify-js` is a more popular drop-in replacement.

fixes #82

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "gzip-size": "^1.0.0",
     "http-server": "^0.7.3",
     "jade": "^1.7.0",
-    "js-size": "^1.0.2",
+    "js-size": "fyockm/js-size#uglify",
```
Comment (fyockm) [inline package.json]: temporary until https://github.com/lukekarrys/js-size/pull/1 is accepted

**Comment by fyockm** [comment]
FWIW...tests actually do pass. I think "saucelabs" is on the sauce.

`Zuul tried to run tests in saucelabs, however no saucelabs credentials were provided.`

**Comment by kamilogorek** [comment]
cc @bear

**Comment by bear** [comment]
@kamilogorek @fyockm  - I've updated the master branch with the proper credentials

**Comment by fyockm** [comment]
thanks @bear. can you re-run travis [on my last commit](https://travis-ci.org/AmpersandJS/amp/builds/50253902) so it shows as passing?

Following that, can I get a +1 on this so it can be merged?

**Comment by bear** [comment]
@fyockm you will need to add the changes I made to .travis.yml to your PR

**Comment by fyockm** [comment]
@bear still failed?

**Comment by bear** [comment]
for whatever odd reason travis-ci is not showing the last commit in this PR so it's failing

**Comment by wraithgar** [comment]
OK builds are passing over in #84 give your +1's there please.

**Pull request merged**